services/web/server/tests/unit/with_dbs/10/meta/test_meta_iterations.py

```diff
@@ -0,0 +1,232 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+
+from http import HTTPStatus
+
+import pytest
+from aiohttp import ClientResponse, web
+from aiohttp.test_utils import TestClient
+from faker import Faker
+from models_library.projects import Project
+from pytest_simcore.helpers.utils_assert import assert_status
+from pytest_simcore.helpers.utils_login import AUserDict
+from pytest_simcore.simcore_webserver_projects_rest_api import (
+    NEW_PROJECT,
+    REPLACE_PROJECT_ON_MODIFIED,
+    RUN_PROJECT,
+)
+from servicelib.json_serialization import json_dumps
+from simcore_postgres_database.models.projects import projects
+from simcore_service_webserver.constants import APP_DB_ENGINE_KEY
+from simcore_service_webserver.director_v2_api import get_project_run_policy
+from simcore_service_webserver.meta_handlers import Page, ProjectIterationAsItem
+from simcore_service_webserver.meta_projects import (
+    meta_project_policy,
+    projects_redirection_middleware,
+)
+from simcore_service_webserver.projects.project_models import ProjectDict
+
+REQUEST_MODEL_POLICY = {
+    "by_alias": True,
+    "exclude_defaults": True,
+    "exclude_none": True,  # e.g. thumbnail: None will fail validation
```

Comment:
thought: sometimes this exclude_none can have significations like, I want to set a value explicitely to None. I prefer the exclude_unset because of this.

Reply:
Agreed, but as you can see, here the problem is the ``project.thumnail``.

The project model, as i have mentioned several times during this review, is not well designed and has been quite a burden in the development of this PR. I took two actions:
- started ``pydantic_model_factory`` together with a collection of examples to test the rationale
- started ``services/web/server/tests/unit/isolated/test_projects__project_models_rest.py`` to safely refactor projects in the new future
- in the meantime, I have added several temporary fixes (marked with FIXME /hack ) to overcome the errors produce when mixing structures containing project data.